bql/planner/planner_test.go

```diff
@@ -272,286 +272,239 @@ func populateStoreWithTriples(ctx context.Context, s storage.Store, gn string, t
 
 func TestPlannerQuery(t *testing.T) {
 	testTable := []struct {
-		q       string
-		wantErr bool
-		nbs     int
-		nrws    int
+		q    string
+		nbs  int
```

Comment:
change this name to numBindings/nBindings?

Reply:
Done.